src/utils/authorizeDelegatedAccount.ts

```diff
@@ -0,0 +1,55 @@
+import { Contract, Wallet, providers } from 'ethers'
+import * as DelegatedAccessRegistry from '../contracts/DelegatedAccessRegistry.sol/DelegatedAccessRegistry.json'
+
+import EthCrypto from 'eth-crypto'
+import handleError from '$/utils/handleError'
+import { Address } from '$/types'
+
+const DelegatedAccessRegistryAddress = '0xf5803cdA6352c515Ee11256EAA547BE8422cC4EE'
+
+export const signDelegatedChallenge = async (
```

Comment:
ATM there's no need to export `signDelegatedChallenge`. Let's not.

There's also no need to make it return anything beyond the signature. Let's do that. Just the signature. Let's keep it simple.